tabular/src/autogluon/tabular/trainer/abstract_trainer.py

```diff
@@ -1432,6 +1491,67 @@ def _get_dummy_stacker(self, level: int, model_levels: dict, use_orig_features=T
         dummy_stacker.initialize(num_classes=self.num_classes)
         return dummy_stacker
 
+    def _proxy_model_feature_prune(self, feature_prune_kwargs: dict, time_limit: float, layer_fit_time: float, level: int, features: List[str]) -> List[str]:
+        """
+        Uses the best LightGBM-based base learner of this layer to perform time-aware permutation feature importance based feature pruning.
+        If all LightGBM models fail, use the model that achieved the highest validation accuracy. Feature pruning gets the smaller of the
```

Comment:
This gets tricky in cases where Image or Text model was the best. Computer Vision models, in particular will 'find' all other tabular features as useless. Shall those be excluded from model candidates?